app/assets/locales/locale-es.json

```diff
@@ -1712,87 +1712,89 @@
     },
     "showcases": {
         "barter": {
-            "account": "Account",
-            "action": "Peer 1 (%(peer_left)s) offers to send %(assets_left)s to Peer 2 (%(peer_right)s) and receives %(assets_right)s in return. ",
-            "add_escrow": "Add escrow",
-            "add_escrow_tooltip": "Adds or removes an escrow to this barter. This escrow will have to approve the resulting proposal as well. The escrow can serve as a witness only, or also be the custodian of Peer 1's funds.",
-            "bartering_asset": "Bartering asset",
+            "account": "Cuenta",
+            "action": "Peer 1 (% (peer_left) s) ofrece enviar% (asset_left) s a Peer 2 (% (peer_right) s) y recibe% (asset_right) s a cambio.",
```

Comment:
`% (asset_left) s` must remain `%(asset_left)s` otherwise it is not recognized as variable